test/test_json/test_project.py

```diff
@@ -89,8 +89,10 @@ def test_project_prod_subp_kpi(self):
         self.signin()
         configs = {
             'WTF_CSRF_ENABLED': True,
-            'PRODUCTS': {'north', 'south'},
-            'SUBPRODUCTS': {'winterfell', 'westeros'}
+            'PRODUCTS_SUBPRODUCTS': {
+                'north': ['winterfell'],
+                'south': ['westeros']
```

Comment:
shouldn't the value for `south` be `['king\'s landing']`?

Reply:
:)